src/game/SpellEffects.cpp

```diff
@@ -1149,6 +1149,13 @@ void Spell::EffectDummy(SpellEffectIndex eff_idx)
 
                     return;
                 }
+                case 32027:                                 // Expedition Flare
+                {
+                    if (!m_CastItem)                        // 32029 = Expedition Preserver | 32030 = Expedition Scout
```

Comment:
Then why won't the spell work if you do a check for m_CastItem with this dummy case then, since this is cast from an item, if our system is not designed so that items are removed before we're done using them?